contrib/native/client/src/clientlib/errmsgs.hpp

```diff
@@ -59,7 +59,8 @@ namespace Drill{
 #define ERR_CONN_SSL_CN         DRILL_ERR_START+27
 #define ERR_CONN_SSL_CERTVERIFY DRILL_ERR_START+28
 #define ERR_CONN_SSL_PROTOVER   DRILL_ERR_START+29
-#define ERR_CONN_MAX            DRILL_ERR_START+29
+#define ERR_CONN_SSL_SNI        DRILL_ERR_START+30
+#define ERR_CONN_MAX            DRILL_ERR_START+30
```

Comment:
Should it be `DRILL_ERR_START+31`?

Reply:
Looking through the history, it looks like ERR_CONN_MAX is always set the value of the highest ERR_CONN code (as in it is an inclusive boundary, not exclusive boundary). So this seems like it should be DRILL_ERR_START+30

Reply:
In that case I'd just add comment for the last entry. Easy to overlook.